src/reference/asciidoc/zookeeper.adoc

```diff
@@ -0,0 +1,57 @@
+[[zookeeper]]
+== Zookeeper Support
+
+=== Introduction
+
+https://zookeeper.apache.org/[Zookeeper] support was added to the framework in _version 4.2_, comprised of:
+
+* MetadataStore
+* LockRegistry
+* Leadership Event Handling
+
+[[zk-metadata-store]]
+=== Zookeeper Metadata Store
+
+The `ZookeeperMetadataStore` can be used where any `MetadataStore` is needed, such as peristent file list filters,
+etc.
+See <<metadata-store>> for more information.
```

Comment:
Any hints here specific to `ZookeeperMetadataStore` and some samples to configure?
BTW have we mentioned in the Docs already the `MetadataStoreListener` support?